babbage/query/cuts.py

```diff
@@ -27,5 +27,5 @@ def apply(self, q, cuts):
             info.append({'ref': ref, 'operator': operator, 'value': value})
             table, column = self.cube.model[ref].bind(self.cube)
             q = self.ensure_table(q, table)
-            q = q.where(column == cast(value, column.type))
```

Comment:
This comment looks like you're drunk!

Shouldn't value be the correct type at this point?